stam.py

```diff
@@ -0,0 +1,49 @@
+import sys
```

Comment:
the documentation for any function should look like this:
```
def my_function( ... ):
   """
   Your documentation here.
   """
```

I recommend you use the **Google-style python docstring** *(it's also the syntax that we use at work)*
check out the link below for more examples:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html